erpnext/manufacturing/report/bom_stock_calculated/bom_stock_calculated.js

```diff
@@ -11,14 +11,14 @@ frappe.query_reports["BOM Stock Calculated"] = {
 			"options": "BOM",
 			"reqd": 1
 		},
-        	{
-	            "fieldname": "qty_to_make",
-        	    "label": __("Quantity to Make"),
-        	    "fieldtype": "Int",
-        	    "default": "1"
-	       },
-
-		 {
+		{
```

Comment:
@kickapoo why is the whitespace messed up??  this is really wasting my review time. So many times we have discussed this. @dottenbr

Reply:
@rmehta I fix the previous commiter indentation issue to follow tabs etc.
Sorry for wasting your time not indention doing so.